plugins/source/test/resources/services/testdata.go

```diff
@@ -0,0 +1,39 @@
+package services
+
+import (
+	"context"
+	"strings"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/testdata"
+)
+
+func TestDataTable() *schema.Table {
+	table := testdata.TestSourceTable("test_testdata_table")
+	for i, c := range table.Columns {
+		if strings.HasPrefix(c.Name, "_cq_") {
+			table.Columns[i].Name = "test" + c.Name
```

Comment:
To reduce complexity, rather than removing the `_cq_id` and `_cq_parent_id` columns here I just add 'test' to the names.